src/components/CompareResults/LinkToRevision.tsx

```diff
@@ -14,6 +14,10 @@ const styles = {
     fontSize: '16px',
     lineHeight: '1.5',
   }),
+  align: style({
```

Comment:
(optional nit) Maybe call that "nowrap" for a more self-describing name :)

```suggestion
  nowrap: style({
```